src/main/java/com/jagrosh/jmusicbot/Bot.java

```diff
@@ -52,38 +52,52 @@
  * @author John Grosh <john.a.grosh@gmail.com>
  */
 public class Bot extends ListenerAdapter {
-    
+
     private final HashMap<String,Settings> settings;
     private final AudioPlayerManager manager;
     private final EventWaiter waiter;
     private final ScheduledExecutorService threadpool;
     private JDA jda;
     private GUI gui;
     //private GuildsPanel panel;
-    public final Category MUSIC = new Category("Music");
-    public final Category DJ = new Category("DJ", event -> 
+    public final Category MUSIC = new Category("Music", event ->
+    {
+        if(event.getAuthor().getId().equals(event.getClient().getOwnerId()))
```

Comment:
These share a lot of common checks with the others. It may be good to factor this out as common functionality.